scripts/check-release.js

```diff
@@ -0,0 +1,90 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const { version: releaseVersion } = require('../lerna.json')
+const { findPackages, printLog, logAndExit, executeCommand } = require('./utils')
+
+async function main() {
+  await checkGitTag()
+  await checkPackages()
+
+  printLog('Release check done.')
+}
+
+async function checkGitTag() {
+  printLog('Checking git tag')
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
  printLog('Checking release version tag is on HEAD')
```